src/Worker/Notifier.php

```diff
@@ -587,7 +587,7 @@ private static function activityPubDelivery($cmd, array $target_item, array $par
 			Logger::log('Deliver ' . $target_item['id'] .' to ' . $inbox .' via ActivityPub', Logger::DEBUG);
 
 			Worker::add(['priority' => $priority, 'created' => $created, 'dont_fork' => true],
-					'APDelivery', $cmd, $target_item['id'], $inbox, $target_item['contact-uid']);
+					'APDelivery', $cmd, $target_item['id'], $inbox, $target_item['contact-uid'] ?: $target_item['uid']);
```

Comment:
Is there no ```$uid``` variable anymore? The ```$uid``` variable had been in use at several other places as well - and they all need this treatment.

Reply:
It used to be in the parameters, I removed it because everything we need is in `$target_item` anyway, but I overlooked this detail. This is the first occurrence I identified to be wrong, there may be more although so far queue is looking better and this fix needs to be merged ASAP.

Reply:
This will most likely fail with posts to Diaspora and Friendica as well. It would be best to have a look at the master and to rework this accordingly. Otherwise we forget about this.

Reply:
Friendica and Diaspora aren't using the UID in this context. I just found two other possibly incorrect uses of UID.